backend/api/v1/auth_service.go

```
@@ -437,37 +438,66 @@ func (s *AuthService) getUserWithLoginRequestOfBytebase(ctx context.Context, req
 func (s *AuthService) getUserWithLoginRequestOfIdentityProvider(ctx context.Context, request *v1pb.LoginRequest) (*store.UserMessage, error) {
 	identityProviderID, err := getIdentityProviderID(request.IdpName)
 	if err != nil {
-		return nil, status.Errorf(codes.InvalidArgument, err.Error())
+		return nil, status.Error(codes.InvalidArgument, err.Error())
 	}
 
 	identityProvider, err := s.store.GetIdentityProvider(ctx, &store.FindIdentityProviderMessage{
 		ResourceID: &identityProviderID,
 	})
 	if err != nil {
-		return nil, status.Errorf(codes.Internal, "failed to get identity provider")
+		return nil, status.Errorf(codes.Internal, "failed to get identity provider: %v", err)
```

The actual error was swallowed, same for other places in this file.